src/modules/info-panel/info-panel.css

```diff
@@ -5,10 +5,14 @@ info-panel {
     position: fixed;
     bottom: 0;
     width: 100%;
-    height: 60%;
+    height: 50%;
     background-color: var(--color-grey);
 }
 
+info-panel.is-maximized {
```

Comment:
could do without the `.info-panel` class, just use the custom tag `info-panel.is-maximized`.